widget/class-widget.php

```diff
@@ -198,7 +216,7 @@ public function update( $new_instance, $old_instance ) {
       $instance = array();
 
       // Update title
-      $instance['title'] = ( ! empty( trim( $new_instance['title'] ) ) ) ? strip_tags( $new_instance['title'] ) : 'Volunteer Opportunities';
+      $instance['title'] = ( ! empty( trim( $new_instance['title'] ) ) ) ? strip_tags( $new_instance['title'] ) : '';
```

Comment:
This caused a fatal error for me since trim() is used inside the empty() function, but empty() is supposed to be passed a variable in older versions of PHP. Check out http://php.net/manual/en/function.empty.php at the note under the Parameters section. I think you can fix this by putting the trim() around the strip_tags() function instead.